charts/incubator/miniflux/values.yaml

```diff
@@ -28,24 +15,46 @@ persistence:
     enabled: true
     mountPath: "/config"
 
-probes:
-  liveness:
+cnpg:
+  main:
     enabled: true
-    custom: true
-    spec:
-      initialDelaySeconds: 30
-      periodSeconds: 10
-      timeoutSeconds: 1
-      failureThreshold: 3
-      httpGet:
-        path: /healthcheck
-        port: 8080
-
-postgresql:
-  enabled: true
-  existingSecret: "dbcreds"
-  postgresqlDatabase: miniflux
-  postgresqlUsername: miniflux
+    user: miniflux
+    database: miniflux
 
 portal:
-  enabled: true
+  open:
+    enabled: true
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    runAsUser: 568
+    runAsGroup: 568
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              enabled: true
+              custom: true
+              spec:
+                initialDelaySeconds: 30
+                periodSeconds: 10
+                timeoutSeconds: 1
+                failureThreshold: 3
+                httpGet:
+                  path: /healthcheck
+                  port: 8080
```

Comment:
This does not exist in the new common. Not sure where did you saw that

Reply:
Ah right, this is from the old common, I forgot to update that part.

Reply:
Updated in 2df4830510a0a251a34b219292c587be1b0e5303.